### ISSUE TITLE
as a forest service employee I want to be able to visually identify a valid xmas tree permit

### ISSUE THREAD

**Issue by sadlerw** [nciinc/fs-intake-module#63]

**Comment by rrefoy** [comment]
@sadlerw how does this differ from #18

**Comment by sadlerw** [comment]
It probably doesn't, but we created this story during the refinement at some point and pointed it.

**Comment by ASprinkle** [comment]
Here is an IDEA of what could be possible for someone to print as a permit.  
I do not have a section for a unique number identifier in this example. I am sharing this as an idea piece-
[XmasTreePermitexample.docx](https://github.com/nciinc/fs-intake-module/files/1515137/XmasTreePermitexample.docx)
 not necessarily something that has to be replicated to a "T".

**Comment by MelissaBraxton** [comment]
TTS is currently planning discovery research around the design of the
print-at-home permit, so we will have some input on this in the coming
weeks. Aiming to wrap up the research the week of 12/18.

Melissa Braxton
User Experience Designer, 18F
Cell: (415) 696-4216
https://18f.gsa.gov/

**Comment by lauraGgit** [comment]
The actual printing of the story has now been moved out to #140.

**Comment by lauraGgit** [comment]
FYI, this doesn't need to necessarily be a PDF, it can just be a print friendly layout.

**Comment by lauraGgit** [comment]
Xmas tree permit view docs in addition to @ASprinkle's template, that may be useful.

- https://github.com/18F/fs-online-permitting/blob/master/docs/christmas-tree-read-out.pdf
- https://github.com/18F/fs-online-permitting/blob/master/docs/christmas-tree-permitting-options.pdf
- https://github.com/18F/fs-online-permitting/blob/master/docs/Christmas-tree-vignettes.pdf

**Comment by bryanmcfadden** [comment]
First draft - Permit proposal

![permit_proposal](https://user-images.githubusercontent.com/16689692/33776637-0b5a56d2-dc10-11e7-881a-4140220e8b9e.png)

**Comment by MelissaBraxton** [comment]
This looks good to me as a first take, and I know we will learn more from the LEO research planned for next week. 
 - One thing - the large black block will require a lot of ink to print, so you might consider how to minimize this. (Thanks to Laura for pointing this out.)

@ASprinkle - If you can take a look and any feedback before Tuesday, 12/12, that'd be great.  I have my first LEO interview on Tuesday, and I'll be bringing a version of this with me for feedback.

**Comment by bryanmcfadden** [comment]
@MelissaBraxton @lauraGgit - I did take the black part in to consideration when the permittee was to print,  but I was trying to create emphasis on the forest location for the permit. After Melissa returns from the LEO research, maybe she will have identified what the KPI and I can adjust accordingly in the design.

**Comment by ASprinkle** [comment]
That is a great draft! it looks very official.  I echo the quantity of Black ink - but also agree that it makes the name very clear - Maybe we could continue with the same gray shade/black ink as the Year...?
Here are my comments (pending results from LEO research)
Where it says to fill in the month and day - it might be helpful to specify  MM and DD next to it so that when these are filled out they are somewhat consistent. The Christmas tree cutting guidelines is an interesting add and I think it does well at getting to some of the important messages. For this block, I would remove one of the 6 inch descriptions. having both in such close proximity is at first confusing to read.  I would remove the 6 inches diameter bit.  I would rephrase the other 6 inch stump bit to read "You must leave a stump of 6 inches or less in height". 
I would add a line that says "Your tree must be 12 feet or less in height" (or whatever the height restriction is for the ARNF)
For the "I have printed rules and regulations" box. It seems like this should be filled in automatically, so an instruction here for the user might be good.
Instead of "Safety is your responsibility" again-  how about a safety tip: "Enjoy your National Forests. Be Safe!" or something fun...? just a thought.

**Comment by MelissaBraxton** [comment]
Applied the design reviewed label for review and approval of this initial draft draft. Refinements handled in #170.
